CleanSlate/common/scripted_effects/00_scripted_effects.txt

```diff
@@ -8207,12 +8207,10 @@ set_correct_feudal_government_type_effect = {
 
 set_correct_theocratic_government_type_effect = {
 	if = {
-		limit = { is_government_potential = muslim_government }
-		set_government_type = muslim_government
-	}
-	else_if = {
-		limit = { is_government_potential = theocratic_feudal_government }
-		set_government_type = theocratic_feudal_government
+		limit = {
+			NOT = { is_government_potential = theocracy_government }
+		}
+		set_government_type = feudal_government
```

Comment:
I think this may need to be migrated to the new custom triggers. I'll look into it.